Source/PTEDashboard.m

```diff
@@ -106,14 +106,13 @@ - (void)handleStatusBarOrientationChange:(NSNotification *)notification
     UIInterfaceOrientation nextOrientation = [notification.userInfo[UIApplicationStatusBarOrientationUserInfoKey] integerValue];
     
     // Flip the window's height and width?
-    CGRect frame = self.frame;
-    if ((UIDeviceOrientationIsLandscape(currentOrientation) && UIDeviceOrientationIsPortrait(nextOrientation)) ||
-        (UIDeviceOrientationIsPortrait(currentOrientation) && UIDeviceOrientationIsLandscape(nextOrientation)))
-    {
-        frame.size = CGSizeMake(frame.size.height,
-                                frame.size.width);
-    }
-    
+	CGRect frame = self.frame;
+	if ((UIInterfaceOrientationIsLandscape(currentOrientation) && UIInterfaceOrientationIsPortrait(nextOrientation)) ||
+		(UIInterfaceOrientationIsPortrait(currentOrientation) && UIInterfaceOrientationIsLandscape(nextOrientation)))
+	{
+		frame.size = CGSizeMake(frame.size.height,frame.size.width);
+	}
```

Comment:
Also you can see [here](https://github.com/PTEz/LumberjackConsole/pull/11/files) how your edits don't look indented in GitHub (and so in many editors that don't represent a tab with four spaces).